builtin/logical/pki/path_issue_sign.go

```diff
@@ -34,6 +35,48 @@ func buildPathIssue(b *backend, pattern string) *framework.Path {
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.UpdateOperation: &framework.PathOperation{
 				Callback: b.metricsWrap("issue", roleRequired, b.pathIssue),
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							"certificate": {
+								Type:        framework.TypeString,
+								Description: `Certificate`,
+								Required:    true,
+							},
+							"issuing_ca": {
+								Type:        framework.TypeString,
+								Description: `Issuing Certificate Authority`,
+								Required:    true,
+							},
+							"ca_chain": {
+								Type:        framework.TypeCommaStringSlice,
+								Description: `Certificate Chain`,
+								Required:    false,
+							},
+							"serial_number": {
+								Type:        framework.TypeString,
+								Description: `Serial Number`,
+								Required:    false,
+							},
+							"expiration": {
+								Type:        framework.TypeString,
```

Comment:
Similar to my feedback in https://github.com/hashicorp/vault/pull/18376, consider using either TypeTime or TypeDurationSeconds here instead.

Reply:
These are unix time. Does it make sense to use `TypeDurationSeconds` here? Idk if it would be properly reflected on the OpenApi side

Reply:
I don't think `TypeDurationSeconds` really applies to absolute Unix time IMO, but maybe someone else disagrees :-)

Reply:
Thanks!